common.mk

```diff
@@ -32,9 +32,47 @@ CCACHE ?= $(shell which ccache) # Don't remove this comment (space is needed)
 # # Set QEMU_VIRTFS_ENABLE to 'y' and adjust QEMU_VIRTFS_HOST_DIR
 # # Then in QEMU, run:
 # # $ mount -t 9p -o trans=virtio host <mount_point>
+# # Or enable QEMU_VIRTFS_AUTOMOUNT
 QEMU_VIRTFS_ENABLE		?= n
 QEMU_VIRTFS_HOST_DIR	?= $(ROOT)
 
+# Persistent Secure Storage via shared folder
+# # Set QEMU_PSS_ENABLE to 'y' and adjust QEMU_PSS_HOST_DIR
+# # Then in QEMU, run:
+# # $ mount -t 9p -o trans=virtio secure /data/tee
+# # Or enable QEMU_PSS_AUTOMOUNT
+QEMU_PSS_ENABLE			?= n
+QEMU_PSS_HOST_DIR		?= $(ROOT)
+
+# Check to avoid virtfs = n and pps = y
+ifneq ($(QEMU_VIRTFS_ENABLE),y)
+ifeq ($(QEMU_PSS_ENABLE),y)
+$(call force,QEMU_VIRTFS_ENABLE,y,required by QEMU_PSS_ENABLE)
+endif
+endif
```

Comment:
I think you can simplify here.
`cal force...` already provides a useful description what's done so no need for comment.
`ifeq ($-QEMU_VIRTFS_ENABLE),y)` not needed.
```patch
-# Check to avoid virtfs = n and pps = y
-ifneq ($(QEMU_VIRTFS_ENABLE),y)
ifeq ($(QEMU_PSS_ENABLE),y)
$(call force,QEMU_VIRTFS_ENABLE,y,required by QEMU_PSS_ENABLE)
endif
-endif
```
And same below:
```patch
-ifneq ($(QEMU_VIRTFS_ENABLE),y)
ifeq ($(QEMU_VIRTFS_AUTOMOUNT),y)
-$(error Virtfs automount is enabled while virtfs disabled; enable virtfs or disable automount)
+$(call force,QEMU_VIRTFS_ENABLE,y,required by QEMU_VIRTFS_AUTOMOUNT)
endif
-endif

-ifneq ($(QEMU_PSS_ENABLE),y)
ifeq ($(QEMU_PSS_AUTOMOUNT),y)
-$(error Persistent secure storage automount is enabled while persistent secure storage disabled; enable persistent secure storage or disable automount)
+$(call force,QEMU_PSS_ENABLE,y,required by QEMU_PSS_AUTOMOUNT)
endif
-endif
```